ServiceApp/src/org/droidplanner/services/android/core/gcs/location/FusedLocation.java

```diff
@@ -41,6 +41,7 @@
 
     private final GoogleApiClientManager gApiMgr;
     private final GoogleApiClientTask requestLocationUpdate;
+    private boolean mLocationUpdatesEnabled = false;
```

Comment:
Hi, In my experience with the Fitness API, GoogleApiClient did seem pretty
picky about the state of things when starting/stopping services. I may be
being a bit paranoid with this, but since the copter is probably flying
around when this happens, it seems warranted. :-)


On Wed, Jun 22, 2016 at 2:03 PM, Fredia Huya-Kouadio <
notifications@github.com> wrote:

> In
> ServiceApp/src/org/droidplanner/services/android/core/gcs/location/FusedLocation.java
> <https://github.com/dronekit/dronekit-android/pull/410#discussion_r68115011>
> :
>
> > @@ -41,6 +41,7 @@
> >
> >      private final GoogleApiClientManager gApiMgr;
> >      private final GoogleApiClientTask requestLocationUpdate;
> > +    private boolean mLocationUpdatesEnabled = false;
>
> @kellyschrock <https://github.com/kellyschrock> is this flag needed? Are
> there adverse effects to calling disableLocationUpdates() if
> enableLocationUpdates() hasn't been called before?
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/dronekit/dronekit-android/pull/410/files/ecc10f24116f5a543bf4dc28efe7f3d5524a6a8c#r68115011>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/ADmhzRjfg858rp8AREmH4UJO3OV-II_Rks5qOYcGgaJpZM4I7YHE>
> .
>